build/build_apple_frameworks.sh

```diff
@@ -189,6 +188,7 @@ cmake_build() {
         ${platform_target:+-DDEPLOYMENT_TARGET=$platform_target} \
         --log-level=VERBOSE
     cmake --build . \
+        -j 6 \
```

Comment:
We're building with xcodebuild here (see `-G Xcode`), so it'll scale the build for us and this options wouldn't have any effect

Reply:
OK, I'll revert this as long as the CI job still takes an hour to finish

Reply:
ah, I checked the logs closer and the PyTorch build is taking 40 minutes so it's unlikely that this is going to help